src/store_engine/dependency_container.ts

```diff
@@ -1,9 +1,14 @@
+import { EventBus } from "../helpers/event_bus";
 import { Get, StoreConstructor, StoreParams } from "./store";
 
+interface StoreUpdateEvent {
+  type: "store-updated";
+}
```

Comment:
TODO: use a separate event instead of `model.trigger('update')`. (It should be independent from `model`)

Reply:
maybe have the bus linked to the storeContainer itself?

Reply:
Yes, that's what I had in mind